python/lsst/pipe/base/testUtils.py

```diff
@@ -362,3 +426,47 @@ def getInitInputs(butler, config):
         initInputs[name] = butler.get(dsType)
 
     return initInputs
+
+
+def lintConnections(connections, *,
+                    checkMissingMultiple=True,
+                    checkUnnecessaryMultiple=True,
+                    ):
+    """Inspect a connections class for common errors.
+
+    These tests are designed to detect misuse of connections features in
+    standard designs. An unusually designed connections class may trigger
+    alerts despite being correctly written; specific checks can be turned off
+    using keywords.
+
+    Parameters
+    ----------
+    connections : `lsst.pipe.base.PipelineTaskConnections`-type
```

Comment:
What does the `-type` signify here? I checked the dev guide and numpydoc guide and saw similar usages but no explanation.

Reply:
It means a `type` object representing that type (or a subtype thereof).